tests/specs/run/sloppy_imports/__test__.jsonc

```diff
@@ -1,10 +1,13 @@
 {
-  "steps": [{
-    "args": "run main.ts",
-    "output": "no_sloppy.out",
-    "exitCode": 1
-  }, {
-    "args": "run --unstable-sloppy-imports main.ts",
-    "output": "sloppy.out"
-  }]
+  "tests": {
+    "no_sloppy": {
+      "args": "run --check main.ts",
+      "output": "no_sloppy.out",
+      "exitCode": 1
+    },
+    "sloppy": {
+      "args": "run --unstable-sloppy-imports --check main.ts",
+      "output": "sloppy.out"
+    }
+  }
```

Comment:
This was during my investigation of the issue. I decided to keep it because it's a better test.